numpy/core/fromnumeric.py

```diff
@@ -861,9 +862,10 @@ def sort(a, axis=-1, kind='quicksort', order=None):
     =========== ======= ============= ============ ========
        kind      speed   worst case    work space   stable
     =========== ======= ============= ============ ========
-    'quicksort'    1     O(n^2)            0          no
-    'mergesort'    2     O(n*log(n))      ~n/2        yes
-    'heapsort'     3     O(n*log(n))       0          no
+    'radixsort'    1     O(n)              n          yes
+    'quicksort'    2     O(n^2)            0          no
+    'mergesort'    3     O(n*log(n))      ~n/2        yes
+    'heapsort'     4     O(n*log(n))       0          no
```

Comment:
Do we need a comment here that `radixsort` is only for a limited subset of dtyes?

Reply:
Done, I thought the version added comment would have been enough, but I made it crystal clear just below the table.